src/OrchardCore.Modules/OrchardCore.Setup/Controllers/SetupController.cs

```diff
@@ -126,6 +127,11 @@ public async Task<ActionResult> IndexPOST(SetupViewModel model)
                 ModelState.AddModelError(nameof(model.RecipeName), S["Invalid recipe."]);
             }
 
+            if (!MailboxAddress.TryParse(model.Email, out var emailAddress))
```

Comment:
IMHO we could have a service for email validation in `OrchardCore.Infrastructure` or `OrchardCore` instead of reference `MimeKit` directly for better organization

If you don't mind I can commit to your PR & fix the failed unit tests if you're busy

Reply:
Good suggestion. `IEmailAddressValidator`, with a default implementation, and then people could decide to change the implementation.

Reply:
Exactly!! Can I modify this pr or start a new one

Reply:
BTW I was partially disconnected for awhile - if not fully - due the a problem with Internet cable in my country

Hope it got fixed soon and start working again